examples/keras_track_distribution.py

```diff
@@ -0,0 +1,56 @@
+import aim
+from aim import track
+
+import numpy as np
+import mnist
+from keras.models import Sequential
+from keras.layers import Dense
+from keras.utils import to_categorical
+
+"""
+Citation for Keras Example Model: victorzhou.com/blog/keras-neural-network-tutorial/
+"""
+
+train_images = mnist.train_images()
+train_labels = mnist.train_labels()
+test_images = mnist.test_images()
+test_labels = mnist.test_labels()
+
+# Normalize the images.
+train_images = (train_images / 255) - 0.5
+test_images = (test_images / 255) - 0.5
+
+# Flatten the images.
+train_images = train_images.reshape((-1, 784))
+test_images = test_images.reshape((-1, 784))
+
+# Build the model.
+model = Sequential([
+  Dense(64, activation='relu', input_shape=(784,)),
+  Dense(64, activation='relu'),
+  Dense(10, activation='softmax'),
+])
+
+# Compile the model.
+model.compile(
+  optimizer='adam',
+  loss='categorical_crossentropy',
+  metrics=['accuracy'],
+)
+
+# Train the model.
+model.fit(
+  train_images,
+  to_categorical(train_labels),
+  epochs=5,
+  batch_size=32,
+)
+
+# Evaluate the model.
+model.evaluate(
+  test_images,
+  to_categorical(test_labels)
+)
+
+weights, biases = model.layers[0].get_weights()
```

Comment:
To track distributions per iteration or per epoch the callback function should be passed to model.fit function which will call `track(aim.weights, model)` from inside. You can learn more about keras callbacks here: https://keras.io/api/callbacks/

Reply:
Hi, I have added a callback class to track weights distributions every epoch.